docs/credentials.md

```diff
@@ -5,12 +5,12 @@ use the MFTF credentials feature to hide sensitive [data][] like integration tok
 
 Currently MFTF supports two types of credential storage: **.credentials file** and **HashiCorp vault**.
 
-# Configure File Storage
+#3 Configure File Storage
 
 The MFTF creates a sample file for credentials during [initial setup][]: `magento2/dev/tests/acceptance/.credentials.example`.
 The file contains an example list of keys for fields that can require credentials.
 
-### Create `.credentials`
+## Create `.credentials`
```

Comment:
The headings are not accurate. For example, "Configure File Storage" should be one level higher than "Create .credentials", etc